plugins/terminal.koplugin/profile

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+# This file gets executed on every start of the KOTerm shell.
+# Do not edit this file!
+# You may change the contents of 'aliases' and 'profile.user'
+#
+
+KOTERM_HOME="$(pwd)"
+# make KOTERM_DATA an absolute path
+# shellcheck disable=SC2164
+cd "${KOTERM_DATA}"
+KOTERM_DATA="$(pwd)"
+HOME="${KOTERM_DATA}"
+
+PATH="${PATH}:${KOTERM_DATA}/scripts:${KOTERM_HOME}/plugins/koterm.koplugin/"
+TERM="vt52"
+
+PS1="$ "
+if [ "$(id -u)" -eq "0" ]; then
+    PS1="# "
+fi
+
+EDITOR="nano"
+
+SHFM_OPENER="${KOTERM_HOME}/plugins/koterm.koplugin/shfm_opener.sh"
+
+export EDITOR
+export HOME
+export KOTERM_HOME
+export KOTERM_DATA
+export PATH
+export PS1
+export SHFM_OPENER
+export TERM
+
+# Source a user profile if it exists.
+USER_PROFILE="${KOTERM_DATA}/scripts/profile.user"
+if [ -f "${USER_PROFILE}" ]; then
+    # shellcheck disable=SC1090
```

Comment:
Somewhat similarly, while we do want to ignore this one, it's better to just use `# shellcheck source=/dev/null`

Shellcheck doesn't serve much purpose if you just disable everything it complains about ;-)